src/EmailObject.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+module.exports = class EmailObject {
+  constructor(emailParams) {
+    this.email = {
```

Comment:
The usage for this `Emailobject.email` looks a bit confusing to me. I mean, it seems like we are using only an 'email' from the whole EmailObject and it's not clear whether it's the sender email or receivers email or something else. Maybe naming this property to something generic like `data`/`payload`/`params` or something similar would make it more clear to understand that it's the whole set of data.

Reply:
Yeah, thanks for suggestions. I also didn't like that email..